sentry/src/main/java/io/sentry/protocol/MeasurementValue.java

```diff
@@ -18,32 +19,31 @@
 @ApiStatus.Internal
```

Comment:
This should not be `@ApiStatus.Internal` anymore I believe.

Reply:
why not?

Reply:
Because it's officially part of the event protocol.

Reply:
hmm, should we also unmark [SentrySpan](https://github.com/getsentry/sentry-java/blob/main/sentry/src/main/java/io/sentry/protocol/SentrySpan.java) and [SentryTransaction](https://github.com/getsentry/sentry-java/blob/main/sentry/src/main/java/io/sentry/protocol/SentryTransaction.java)?  They are also part of the documented protocol

Reply:
I guess so, I'd not block this PR for this tho.
Just mentioned because it touches this file specifically, it was an internal API and now it's not anymore.
We likely need to remove the internals from SDKs now (related to performance), Dart should have some as well.

Reply:
But these classes are not part of the user facing API are they? They're "internal" to the SDK. Shouldn't that imply the `@ApiStatus.Internal`?

Reply:
`SentryTransaction` is used in the EventProcessor for example, so it's technically not Internal. Same with `SentrySpan` and `MeasurementValue` if we unmark `SentryTransaction` internal.

I think we should generally revisit the public api of the sdk

Reply:
Thanks for the clarification. So I guess the plan is to remove all of the annotations in one PR and not remove it for `MeasurementValue` here?

Reply:
yeah, I'll create an issue for that

Reply:
https://github.com/getsentry/team-mobile/issues/57